blob/service_test.go

```diff
@@ -492,6 +496,66 @@ func TestService_GetAllWithoutPadding(t *testing.T) {
 	}
 }
 
+// TestRetrieveBlobsFromRealEds takes the real data, submitted to mocha-4 and
+// ensures that all blobs under `0x51cc1a73d65f95f6bde6` namespace will be retrieved successfully.
+// At the first step it retrieves all blobs under the given namespace:
+// the amount of the blobs is known and equal to 6. Then it ensures that each blob has a correct index inside the eds
+// by requesting share and comparing them.
+func TestRetrieveBlobsFromRealEds(t *testing.T) {
+	f, err := os.OpenFile("example/eds.json", os.O_RDONLY, os.ModePerm)
```

Comment:
Example name will be confusing. People will think thats an example on how to use blob module there. It should be called smth like testdata instead.

Reply:
removed this test